packages/react-next/release.md

```diff
@@ -2,7 +2,33 @@
 
 ## Breaking changes
 
+### Checkbox
+
+- Removed `styles` prop.
+- Removed `checkmarkIconProps` prop.
+- Deprecated onRenderLabel.
+- Added label/checkmark slot props.
+
 ### Pivot
 
 - Converted Pivot to a function component.
```

Comment:
Probably good to remove since we're already talking about the "Function component conversion" below.